integration_test/gce/gce_testing.go

```diff
@@ -1500,57 +1500,66 @@ func InstallGsutilIfNeeded(ctx context.Context, logger *log.Logger, vm *VM) erro
 		return fmt.Errorf("this test does not know how to install gsutil on platform %q", vm.Platform)
 	}
 
-	// This is what's used on openSUSE.
-	repoSetupCmd := "sudo zypper --non-interactive refresh"
-
-	if strings.HasPrefix(vm.Platform, "sles-") {
-		// Use a vendored repo to reduce flakiness of the external repos.
-		// See http://go/sdi/releases/build-test-release/vendored for details.
-		repoArch := "x86-64"
-		if IsARM(vm.Platform) {
-			repoArch = "aarch64"
+	gcloudArch := "x86_64"
+	if IsARM(vm.Platform) {
+		gcloudArch = "arm"
+	}
+	gcloudPkg := "google-cloud-cli-453.0.0-linux-" + gcloudArch + ".tar.gz"
```

Comment:
Do they maintain a direct link to the latest tarball? If not, is there a way to retrieve the latest version programmatically?
We might be adding yet another chore to periodically upgrade gcloud on SLES/OpenSUSE or let that setup diverge from other distros.

Reply:
Good point! Though I couldn't find a direct link for the tarball installation method. Modified the script to run `gcloud components update` after the installation to keep the cli to the latest version. I think alternatively we could also use the installer method to download the latest version as before, and use the VERSION there to download the latest tarball.